.travis.yml

```diff
@@ -12,3 +12,7 @@ script:
   ./gradlew check --info
 after_success:
   - bash <(curl -s https://codecov.io/bash)
+# Workaround for Travis CI issue 5227 causing buffer overflow in OpenJDK7.
+before_install:
+    - sudo hostname "$(hostname | cut -c1-63)"
```

Comment:
forgive my ignorance, but how do we sudo when the first lines says `sudo: false`? Does the restriction only kick in after installation or something?

Reply:
I honestly don't know. I switched to using the hosts/addons fix so we don't have to worry about sudo usage.